src/Components/Blazor/WebAssembly.Authentication/src/Interop/AuthenticationService.ts

```diff
@@ -0,0 +1,319 @@
+import { UserManager, UserManagerSettings, User } from 'oidc-client'
+
```

Comment:
For this file as a whole, is there any intention that people can consume it as a library from their other TypeScript code, or should it be considered a black-box implementation detail for the .NET code?

In other words, do we need to API-review this as if it was public API, or are we treating it all as private?